pennylane/devices/default_qubit.py

```diff
@@ -249,6 +260,19 @@ def _get_batch_size(self, tensor, expected_shape, expected_size):
 
     # pylint: disable=arguments-differ
     def apply(self, operations, rotations=None, **kwargs):
+        """Apply a sequence of operations and rotations to the device.
+
+        .. warning::
+
+            Directly calling this method is not recommended and should be avoided where possible.
+            The available methods of DefaultQubit will be updated soon to follow a new API described
+            `here <https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.experimental.DefaultQubit2.html>`_.
+
+        Args:
+            operations (List[.Operation]): List of operations to apply
+            rotations (List[.Operation]): List of rotations to apply to rotate state to measurement basis
+        """
```

Comment:
These are the docstrings already available for [apply](https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.default_qubit.DefaultQubit.html#pennylane.devices.default_qubit.DefaultQubit.apply):
![image](https://github.com/PennyLaneAI/pennylane/assets/49409390/ee77a1ed-6430-46e9-9432-6fdf05c98cd1)

Have you shortened them consciously to dissuade users from using this method?